django/contrib/auth/password_validation.py

```diff
@@ -0,0 +1,134 @@
+from __future__ import unicode_literals
+
+import os
+import re
+from difflib import SequenceMatcher
+
+from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured, ValidationError
+from django.template.defaultfilters import unordered_list
+from django.utils.encoding import force_text
+from django.utils.module_loading import import_string
+from django.utils.six import string_types
+from django.utils.translation import ugettext as _
+
+
+def get_password_validators():
+    validators = []
+    for validator in settings.AUTH_PASSWORD_VALIDATORS:
+        try:
+            klass = import_string(validator['NAME'])
+            validators.append(klass(**validator.get('OPTIONS', {})))
+        except ImportError:
+            raise ImproperlyConfigured("Invalid NAME for a password validator: %s. Check "
+                                       "your AUTH_PASSWORD_VALIDATORS setting." % validator['NAME'])
+
+    return validators
+
+
+def validate_password(password, user=None):
+    """
+    Validates whether the password meets all validator requirements.
+
+    If the password is valid, returns ``None``.
+    If the password is invalid, raises ValidationError with all error messages.
+    """
+    errors = []
+    for validator in get_password_validators():
+        error = validator.validate(password, user)
+        if error:
+            errors.append(error)
+    if errors:
+        raise ValidationError(errors)
+
+
+def password_changed(password, user=None):
+    """
+    Informs all validators, that have implemented this feature, that the password has been changed.
+    """
+    for validator in get_password_validators():
+        if hasattr(validator, 'password_changed'):
+            validator.password_changed(password, user)
+
+
+def password_validators_help_texts():
+    """
+    Returns a list of all help texts of all configured validators.
+    """
+    help_texts = []
+    for validator in get_password_validators():
+        help_texts.append(validator.get_help_text())
+    return help_texts
+
+
+def password_validators_help_text_html():
+    """
+    Returns an HTML string with all help texts of all configured validators in an <ul>.
```

Comment:
I would read it as "validators in a 'U' 'L'" so I would say "in a" rather than "in an" but if you read it as "unordered list" than "an" is fine.